### ISSUE TITLE
Doesn't work on front end

### ISSUE THREAD

**Issue by Hube2** [Hube2/acf-custom-field-locations-rule#1]
Does not work on front end with acf_form(). Not sure it it can be corrected. Will look into it.

**Comment by vitalijm** [comment]
Does it work for know on frontend?

**Comment by Hube2** [comment]
Yep, this is on my to do list, not really sure why it's not loading or working on the front end.
